be/src/vec/sink/vdata_stream_sender.cpp

```diff
@@ -629,7 +630,8 @@ Status VDataStreamSender::send(RuntimeState* state, Block* block, bool eos) {
                 std::vector<SipHash> siphashs(rows);
                 // result[j] means column index, i means rows index
                 for (int j = 0; j < result_size; ++j) {
-                    block->get_by_position(result[j]).column->update_hashes_with_value(siphashs);
+                    unpack_if_const(block->get_by_position(result[j]).column)
```

Comment:
add some comment here, to explain whey do we need uppack const column.

Reply:
done!